app/controllers/people_controller.rb

```diff
@@ -0,0 +1,14 @@
+class PeopleController < ApplicationController
+  def index
+    @people = Person.all
+  end
+
+  def show
+    @person = Person.find(params[:id])
+  end
+
+  def tasks
+    @person = Person.find(params[:id])
+    @tasks = Task.all
+  end
```

Comment:
Since this is in the people controller, I might name this method "task_by_person" to make it extra clear.